src/protocols/rdp/Makefile.am

```diff
@@ -224,6 +224,7 @@ BUILT_SOURCES =                         \
 rdp_keymaps =                                \
     $(srcdir)/keymaps/base.keymap            \
     $(srcdir)/keymaps/failsafe.keymap        \
+    $(srcdir)/keymaps/da_dk_qwerty.keymap    \
```

Comment:
Is this from something else?

Reply:
No, I just figured I'd order the rest of the keymaps in the code block while I put the Norwegian one in its correct place. On the contrary I was unsure about whether to move `failsafe.keymap` to its alphabetical position and decided against it, since it could be argued to be a special case that perhaps deserves special visibility. As you see, I just moved the Danish keymap from directly below the Swedish to its more alphabetically correct position above the German one.

Reply:
Ah, OK. I didn't see the deletion below. I thought this was adding a keymap that didn't yet exist.